ydb/core/protos/config.proto

```diff
@@ -1325,9 +1325,9 @@ message THiveConfig {
     }
 
     enum EHiveChannelBalanceStrategy {
-        HIVE_CHANNEL_BALANCE_STRATEGY_HEAVIEST = 1;
-        HIVE_CHANNEL_BALANCE_STRATEGY_RANDOM = 2;
-        HIVE_CHANNEL_BALANCE_STRATEGY_WEIGHTED_RANDOM = 3;
+        HIVE_CHANNEL_BALANCE_STRATEGY_HEAVIEST = 0;
+        HIVE_CHANNEL_BALANCE_STRATEGY_RANDOM = 1;
+        HIVE_CHANNEL_BALANCE_STRATEGY_WEIGHTED_RANDOM = 2;
```

Comment:
This introduces compatibility issues with config versions?

Reply:
It does, but this is the PR that adds the ability to use this enum through web ui. The enum is not in any of stable releases, and I don't believe anyone has a yaml-config with it anywhere at this point. 
(I changed it because it turns out C++ proto enum descriptors assume that enum values are consecutive and start with zero)